libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -871,6 +900,10 @@ void AP_Baro::update(void)
             if (sensors[i].type == BARO_TYPE_AIR) {
 #if HAL_BARO_WIND_COMP_ENABLED
                 corrected_pressure -= wind_pressure_correction(i);
+#endif
+#if AP_BARO_THST_COMP_ENABLED
+                corrected_pressure -= thrust_pressure_correction(i);
+                sensors[i].corrected_pressure = corrected_pressure;
```

Comment:
This additional logging should be enabled for wind comp or thrust comp. 
```suggestion
#endif
#if (HAL_BARO_WIND_COMP_ENABLED || AP_BARO_THST_COMP_ENABLED)
                sensors[i].corrected_pressure = corrected_pressure;
```

Reply:
I wonder if it would be more intuitive to log the correction itself rather than the corrected value. Would make it easier to look at of the course of a log. Could calculate the difference with MAVExplorer I guess.

Reply:
I fixed the #define. Don't know about the logging - I feel its better this way as you can easily see the pressure that is actually being used.